### DISCUSSION TITLE
Remove the Most Helpful widget

### DISCUSSION THREAD

**Discussion by ahundiak** [symfony/symfony#43007] [Q&A] [answered]
I'd like to suggest removing the most helpful widget assuming it is easy to do so.

I have watched the list change over the last month or so.  Often wondering why the changes were made as users moved up and down.  As of this particular posting my username is at the top of the list!  Not because I have been the most helpful but rather because it is alphabetical in nature.

In any event, the slack channel never needed any kind of rep indicator and it seemed to work fine.  I just don't see the point of the widget and I suspect the community does not want this site to become rep driven.

**Comment by mbabker** [comment, marked as answer]
It can't be done, that's a GitHub feature and there are no settings for it.
